src/sorcha/activity/identity_activity.py

```diff
@@ -0,0 +1,55 @@
+from sorcha.activity.base_activity import AbstractCometaryActivity
+import pandas as pd
+
+"""
+!!!!!!!!!!!!!!!!
+FOR TESTING ONLY
+!!!!!!!!!!!!!!!!
+"""
+
+
+class IdentityCometaryActivity(AbstractCometaryActivity):
+    """
+    !!! THIS SHOULD NEVER BE USED - FOR TESTING ONLY !!!
+
+    Rudimentary cometary activity model that returns no change to the input ``observation``
+    dataframe.
+    This class is explicitly created for testing purposes.
+    """
+
+    def __init__(self) -> None:
+        super().__init__()
+
+    def compute(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Returns numpy array of 0's with shape equal to the input dataframe
+        time column.
+
+        Parameters
+        ----------
+        df : pd.DataFrame
+            The ``observations`` dataframe provided by ``Sorcha``.
+
+        Returns
+        -------
+        pd.DataFrame
+            The original ``observations`` dataframe, unchanged.
+        """
+
+        self._validate_column_names(df)
+
+        return df
+
+    @staticmethod
+    def name_id() -> str:
```

Comment:
I think this could be changed to use the `@property` decorator

```suggestion
    @property
    def name_id() -> str:
```

that way we can access the `name_id` in a more pythonic way, like `activity.name_id`

Reply:
aaaah I see, I assumed that we were instantiating the subclass, but if that's not the case then `@staticmethod` should work fine!

Reply:
Cool. Thanks a bunch for your help, I really appreciate being about to talk through these ideas with you 😃